nose_parameterized/parameterized.py

```diff
@@ -211,11 +211,53 @@ def expand(cls, input, testcase_func_name=None):
             >>>
             """
 
-        def parameterized_expand_wrapper(f):
+        def doc_expansion(f, args, kwargs, is_method):
+            offset = 1 if is_method else 0
+            vns = inspect.getargspec(f)[0][offset:]
+            descs = ["{0} = {1}".format(n, repr(v)) for n, v in zip(vns, args)]
```

Comment:
Instead of using straight repr here, can we get a bit fancy and add a `short_repr` function, which would basically be:

```
def short_repr(x, ln=64):
    x_repr = repr(x)
    if len(x_repr) > n:
        x_repr = x_repr[:n/2] + "..." + x_repr[len(x_repr) - n/2:]
    return x_repr